### PR TITLE
Appdev 9354 add size to batch transfer

### PR DESCRIPTION
**Pull request by sharonluong**
This PR does multiple things:

- cleans up extensive duplication between AV items import, batch transfer audio import, and batch transfer video import
- fixes CSS sass compiling issues
- add size column to batch transfer audio import

### CODE REVIEW

**Comment by chrisrlc** [comment]
Could possibly add how to compile assets in the README?

**Comment by sharonluong** [comment]
> Could possibly add how to compile assets in the README?

Ah, good idea!

File: app/Http/Controllers/TransfersController.php [hunk 1]
```diff
@@ -316,9 +316,11 @@ public function audioImportUpload(Request $request)
  
       $import = new AudioImport($filePath);
       $data = $import->data();
+      $possibleDataKeys = Transfer::AUDIO_IMPORT_KEYS;
+      $tableId = 'audio-upload-data-container';
```
Comment (chrisrlc) [inline app/Http/Controllers/TransfersController.php:320]: I think the $tableId vars all follow the same format in each of the controllers, so if you wanted, you could pass in just "audio" or "items" to the partial and render "{{dataType}}-upload-data-container" in the view, but that may be overkill/maybe you have exceptions to this format.

Comment (chrisrlc) [reply app/Http/Controllers/TransfersController.php:320]: Same with $divId, I think

Comment (sharonluong) [reply app/Http/Controllers/TransfersController.php:320]: 👍

**Comment by chrisrlc** [comment]
The changes look good to me! I don't know much about laravel/php tests, but they seem sensible!

**Pull request merged**